Tests/MongoSwiftSyncTests/CrudTests.swift

```diff
@@ -181,6 +181,7 @@ private class CrudTest {
     var arrayFilters: [BSONDocument]? { self.args["arrayFilters"]?.arrayValue?.compactMap { $0.documentValue } }
     var batchSize: Int? { self.args["batchSize"]?.toInt() }
     var collation: BSONDocument? { self.args["collation"]?.documentValue }
+    var comment: BSON? { self.args["comment"] }
```

Comment:
unless I'm mistaken it looks like all of the new tests you've synced in are in the unified test format? this test file is for the "legacy" crud format which no new tests are going to be written in. so I don't think we need any of the changes in this file.

Reply:
I think so too, I just ctrl+f'd `options` and made sure each instance had comment. Restored to the original and removed all changes in this file. The CRUD unified tests still seem to pass so you're prob right that my only test changes were unified